Mohammed/index.js

```diff
@@ -0,0 +1,23 @@
+
+let mainStr = "It's hard to say who is right in this matter";
+
+// 0.
+let firstOccurrence = 0;
+lastIndexOf_a = mainStr.lastIndexOf('a');
+/* console.log(lastIndexOf_a); */
+lastIndexOf_aExtraction = mainStr[lastIndexOf_a];
+console.log(lastIndexOf_aExtraction);
```

Comment:
Wrong answer! Try this one instead!
console.log(mainStr.indexOf("a", 17)); //39